example/configs/simple_example.yaml

```diff
@@ -0,0 +1,19 @@
+hf_configuration:
+  hf_dataset_name: yourbench_example # change this to your desired dataset name
+  private: false # change this to true if you want to make the dataset private. it's true by default.
+
+model_list:
+  - model_name: Qwen/Qwen3-30B-A3B
+    provider: fireworks-ai
+
+pipeline:
+  ingestion:
+    source_documents_dir: example/data/raw
+    output_dir: example/data/processed
+  upload_ingest_to_hub:
```

Comment:
add some doc, are the values expected booleans?

Reply:
not sure i get what you mean... 

(this example will run standalone. my motivation was to give users a low LoC file that they can run as a PoC with just a `HF_TOKEN` and they can look at the `advanced_example.yaml` after. i'll add some comments / leave a link to the advanced config / FAQs)